doc/manual-installation.md

```diff
@@ -72,9 +72,7 @@ metadata:
 spec:
   version: 0.4.0
```

Comment:
It may be good to change this to **0.7.0** to be consistent.

Reply:
done